routes/routes.js

```diff
@@ -8,4 +8,20 @@ routes.get('/hello', (req, res) => {
 routes.get('/hello/:name', (req, res) => {
     res.send(`Hello, ${req.params.name}`);
 });
-module.exports = routes;
\ No newline at end of file
+
+routes.get('/animals', (req, res) => {
+    const species = ['elephant', 'cat', 'dog', 'snake', 'bee', 'bird'];
+    const getRandomSpecies = () => {
+        return species[Math.floor(Math.random() * species.length)];
+    };
+    const speciesToShow = [getRandomSpecies()];
+    for (let i = 0 ; i < req.query.amount-1 ; i++) {
+        speciesToShow.push(getRandomSpecies());
+    }
```

Comment:
Makes no sense to add one initial element (logic is the same!). Just create an empty array and fill it.

Reply:
Even if the amount is undetermined in url (e.g. http://localhost:3000/animals) server have to return 1 random value. That was the aim. So in this example req.query.amount will be undefined and this loop won't work.

Reply:
If you need to have at least one element, this indeed've sense.

Reply:
:+1: